changelogs/fragments/177-remove_tests_sanity_requirements.yml

```diff
@@ -0,0 +1,3 @@
+---
+trivial:
+  - sanity tests - remove unnecessary requirements.txt file (https://github.com/ansible-collections/overview/issues/43).
```

Comment:
I don't think a changelog fragment is needed for this. I'd have one for the galaxy.yml change though, except if that change was only added in `main` and hasn't been released yet.

Reply:
@felixfontein it would fail otherwise. This collection requires even `trivial` one

Reply:
Then you should probably also mention the galaxy.yml change.

Reply:
Done!